mailpoet/lib/Form/Widget.php

```diff
@@ -262,6 +265,7 @@ public function widget($args, $instance = null) {
       // render form
       $renderer = (new RendererFactory())->getRenderer();
       try {
+        $data = $this->wp->applyFilters('mailpoet_form_widget_pre_process', $data);
```

Comment:
Could you please add test, similarly to how [`mailpoet_form_widget_post_process` is tested](https://github.com/mailpoet/mailpoet/blob/4ea8a51d1dccfbdee74f634dc804c6b1390454b7/mailpoet/tests/integration/Form/WidgetTest.php#L38-L54)?